app/presenters/guide_presenter.rb

```diff
@@ -50,6 +50,7 @@ def details
     details_hash = {
       body: govspeak_body.to_html,
       header_links: level_two_headers,
+      change_history: GuidePresenter::ChangeHistoryPresenter.new(guide, edition).change_history
```

Comment:
There shouldn't be a need for `GuidePresenter::` in the above because `ChangeHistoryPresenter` is in the namespace.